src/lib/update-alternatives/model/alternative.rb

```diff
@@ -0,0 +1,54 @@
+require "cheetah"
+
+module UpdateAlternatives
+  # Represents an alternative
+  class Alternative
+    attr_reader :name
+    attr_reader :status
+    attr_reader :value
+    attr_reader :choices
+    Choice = Struct.new(:path, :priority, :slaves)
```

Comment:
looks good now...btw as whole Alternative also do not have any instance methods, you can use
```
class Alternative < Struct.new(:name, :status, :value, :choices)
  Choice = Struct.new(:path, :priority, :slaves)
  
   ...
end